cpp/src/Slice/Parser.h

```diff
@@ -702,16 +705,17 @@ class Operation : public virtual Contained, public virtual Container
     size_t returnBitSequenceSize() const;
 
     void destroy() override;
-    TypePtr returnType() const;
-    bool returnIsTagged() const;
-    int returnTag() const;
+    TypePtr returnType() const; //TODO remove this once the compilers have been updated to use return-tuples.
```

Comment:
Should rename inBitSequenceSize to paramsBitSequenceSize.

Reply:
Fixed.